web/testing/cypress/e2e/ID005_Edit_Unique_Event.cy.js

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable no-undef */
+/// <reference types="cypress" />
+
+describe("Edit Unique Events", () => {
+  beforeEach(() => {
+    cy.viewport(1440, 900);
+
+    cy.visit("/");
+  });
+
+it("Edit an unique event", () => {
+  //log in
+  const userEmail = `e2e+${Math.ceil(
+    Math.random() * 1000000000000000
+  )}@testdfas.com`;
+  console.log(userEmail);
+  cy.get(".w-full > .flex").click();
+  cy.origin(
+    "https://oncampus.us.auth0.com",
+    { args: { userEmail } },
+    ({ userEmail }) => {
+      cy.get("a").contains("Sign up").click();
+      cy.get("input#email").type(userEmail); //me might have to go through signup every time and create a new account? might trigger the auth0 limits if we do that...
+      cy.get("input#password").type("#e5PUJe9@mroL$", { log: false });
+      cy.contains("button[value=default]", "Continue").click();
+      cy.get("button").contains("Accept").click();
+    }
+  );
+
+  //create volunteer event
+  cy.visit("/schedule");
+  cy.wait(10000);
```

Comment:
just so you know cy.wait is a bad practice (in this case whatever works, works that's why I mentioned using it). Cypress automatically waits for page loads so if the schedule is not loading in time you can increase the timeout instead of using a wait. If there is an element that needs to load in order to continue you can do cy.get(element) and it will wait for it to load, using a timeout which is the right way to wait. The main reason is that on large scale projects the test can take ages to run and the wait is an unreductible amount of time, while the timeout will take the minimum time required.

Reply:
good to know, thanks!